pages/common/apkeep.md

```diff
@@ -0,0 +1,16 @@
+# apkeep
+
+> Download APK files from various sources.
+> More information: <https://github.com/EFForg/apkeep>.
+
+- Download an APK:
```

Comment:
```suggestion
- Download an APK file to the specified directory:
```

Is this correct? Otherwise the second placeholder remains unexplained.

Reply:
Yeah.